includes/Classifai/Command/RSSImporterCommand.php

```diff
@@ -204,7 +206,7 @@ public function get_url_meta( $url ) {
 		$options['timeout']              = 60; // phpcs:ignore WordPressVIPMinimum.Performance.RemoteRequestTimeout.timeout_timeout
 
 		$request_url = 'https://mercury.postlight.com/parser?url=' . rawurlencode( $url );
-		$response    = wp_remote_get( $request_url, $options ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get
+		$response    = safe_wp_remote_get( $request_url, $options ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get
```

Comment:
Can remove this ignore statement now
```suggestion
		$response    = safe_wp_remote_get( $request_url, $options );
```